### ISSUE TITLE
BIG issue: Save delete in MP.

### ISSUE THREAD

**Issue by MisterTimLol** [loki79uk/FS25_UniversalAutoload#70]
Latest deletes savegame in multiplayer!

**Comment by loki79uk** [comment]
Is this player hosted or dedicated server?

**Comment by loki79uk** [comment]
Duplicate of #57 

same issue of renaming mod zip file:
```
Available mod: (Hash: 645e5222d7b00a5ef3d5a0981a1dd96b) (Version: 0.0.0.10) FS25_UniversalAutoload_0_0_0_10
```
causing xml error:
```
2024-12-03 19:53 dataS/character/playerM/facialHair/mHead01/upwardHandlebar.i3d (4.66 ms)
2024-12-03 20:22 Error: Running LUA method 'packetReceived'.
dataS/scripts/xml/XMLFile.lua:537: invalid use of '%' in replacement string
2024-12-03 20:24 Error: Running LUA method 'packetReceived'.
dataS/scripts/xml/XMLFile.lua:537: invalid use of '%' in replacement string
2024-12-03 20:24 Error: Running LUA method 'packetReceived'.
dataS/scripts/xml/XMLFile.lua:537: invalid use of '%' in replacement string
```
